### ISSUE TITLE
Getting Started - Can't get demo code to run

### ISSUE THREAD

**Issue by ashwaths0793** [react-bootstrap-table/react-bootstrap-table2#223]
I'm really sorry to bother. I'm trying to understand React-Bootstrap-Table-2 and I am having a lot of trouble getting the demos to work. I was hoping you could help me solve my problem. I keep getting an error - 

bundle.js:2356 Warning: Caption(...): No `render` method found on the returned component instance: you may have forgotten to define `render`, returned null/false from a stateless component, or tried to render an element whose type is a function that isn't a React component.

My application consists of one component and your demo code (see below). 

import React, {Component} from 'react';
import BootstrapTable from 'react-bootstrap-table-next';

class RenderTable extends Component {
    

    render (){
        const products = [{id:"1", name:"Doe", price:"46"}];
        const columns = [{
            dataField: 'id',
            text: 'Product ID'
        }, {
            dataField: 'name',
            text: 'Product Name'
        }, {
            dataField: 'price',
            text: 'Product Price'
        }];

        return (
            <div>
                <BootstrapTable keyField='id' data={ products } columns={ columns } />
            </div>

        );

    }

}

export default RenderTable;

**Comment by AllenFang** [comment]
@ashwaths0793 sorry for lately reply, I cant reproduce this issue, could you please give a workable example or a repo. Thanks

BTW, What react version you used?

**Comment by ashwaths0793** [comment]
Hi Allen, 

Thank you so much for response. I uploaded my project to a git repository (https://github.com/MULeverLab/ReduxSimpleStarter-LeverUI). There are several components, but the one where I am trying to render a bootstrap table (The code that I sent earlier) is in **src/components/table_bootstrap.js**. The application uses react-router and I try to render this component in the path localhost:8080/table (which is declared in the file **src/index.js**). 

I think the problem might be how I import the react-bootstrap-table2 styles. I was unsure where to put this line (import 'react-bootstrap-table-next/dist/react-bootstrap-table2.min.css';) that you had specified on your documentation. I am new to react and any help is great. Thank you in advance for everything. 

Thanks, 
Ash

**Comment by AllenFang** [comment]
@ashwaths0793 upgrade the react to newest and try again.

**Comment by ashwaths0793** [comment]
Will do. Thank you!

**Comment by ashwaths0793** [comment]
Thank you Allen for all the help. I got it to work! You were right about the version of react being the problem. I was using a boiler plate starter that was recommended by a course instructor. For some reason I was unable to update/upgrade the react version (bunch of errors appeared following the directions on the React documentation). I decided to recreate the application using the starter found on the React website and your library works great. I cannot that you enough for all the help. You were a great help and your library is fantastic. Have a wonderful weekend.
